docs/DesignDocs/Attestation_API_Proposal.md

```diff
@@ -591,63 +594,119 @@ plugin libraries, composed of:
   - It also implements OCALLs for the enclave-side library to access
   host-side services for SGX quote verification.
 
-The OE SDK V0.10 release will also have enclave-side and host-side plugins
-for verification of SGX ECDSA reports generated by the legacy API function
-`oe_get_report()` and raw SGX ECDSA quotes generated by the Intel SGX SDK.
-These two evidence formats are identified by their UUIDs:
-- `OE_FORMAT_UUID_SGX_ECDSA_P256_REPORT`: for SGX ECDSA report generated
-by `oe_get_report()`.
-- `OE_FORMAT_UUID_SGX_ECDSA_P256_QUOTE`: for raw SGX ECDSA quote generated
-by the Intel SGX SDK.
+Currently there is no plan to implement verifier plugins for verification of
+EPID evidence. Verification of SGX EPID quotes is not supported by
+the Intel SGX SDK. In all existing SGX EPID solutions, EPID quotes are sent to
+Intel Attestation Service (IAS) backend for verification.
```

Comment:
Can you point to an SDK or doc that explains how to send them to the IAS service?

Reply:
Will add some links. Thanks!